repository/w.json

```diff
@@ -468,6 +468,17 @@
 				}
 			]
 		},
+		{
+			"name": "Wikidot Syntax highlighter",
+			"details": "https://github.com/MrAbadede/sublime-wikidot-syntax-highlighting",
+			"labels": ["syntax", "highlight", "wikidot"],
+			"releases": [
+				{
+					"sublime_text": ">=4000",
+					"branch": "master"
```

Comment:
Branch based releases are not supported. Please set this to `"tags": true` and tag a release using proper semver version numbering.